components/_twig-components/functions/bem.function.php

```diff
@@ -0,0 +1,42 @@
+<?php
```

Comment:
Will we ever update it? And if so, do we want people to be aware of the update?

I'm thinking:
If we won't update it, git ignore it
If we will update it and git ignore it, they won't know there's an update
If we will update it and they commit it, they'll see the update when they do a git diff

Not sure what's right, just laying out the possibilities I see